docs/distributed/README.md

```diff
@@ -1,74 +1,85 @@
 # Distributed Minio Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-Minio in distributed mode lets you pool multiple drives (even on different machines) into a single object storage server. As drives are distributed across several nodes, distributed Minio can withstand multiple node failures and yet ensure full data protection.
+When running Minio in distributed mode (_distributed Minio_), multiple drives from different machines can be pooled into a single object storage server. As drives are distributed across several nodes, distributed Minio ensures full data protection when multiple node failures occur.
```

Comment:
"into a single storage server" might be confusing for some users, maybe single namespace is better here? @deekoder

Reply:
Per conversation with @deekoder this is fine